partialWaveFit/fitResult.cc

```diff
@@ -343,6 +342,14 @@ fitResult::fill(const fitResult& result)
 	_hasHessian             = result.hasHessian();
 }
 
+multibinCenterType fitResult::multibinCenter() const
+{
+	multibinCenterType multibinCenter;
+	for (binningMapType::const_iterator it = _binningMap.begin(); it != _binningMap.end(); ++it) {
+		multibinCenter[it->first] = 0.5 * (it->second.first + it->second.second);
+	}
+	return multibinCenter;
+}
```

Comment:
functions should be surrounded by two empty lines, the return type of a function should be on a separate line.